### PR TITLE
IDF Plugin docs

### PR DESCRIPTION
**Pull request by pablocarle**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/cli-idfplugin.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+At this stage, the Plug-in does not interface with the mainframe system, so no Zowe CLI profile configuration is required.
+
+Use `zowe idf map --help` to get the most up-to-date details of the required parameters.
+
+`zowe idf map "$CSV" --esm "$ESM" --system "$SYSTEM" --registry "$REGISTRY"`
+
+- `$CSV`: Path to the input CSV-formatted file, see below for the details of the format.
+- `$ESM`: Identifier of the target ESM, one of ACF2, TSS or RACF.
+- `$SYSTEM`: Optional parameter, system identifier for JCL purposes.
+- `REGISTRY`: Registry to identify the distributed identity provider, for example LDAP `ldap://12.34.56.78:389`
```
Comment (JirkaAichler) [inline docs/user-guide/cli-idfplugin.md:46]: Should it be $REGISTRY? Why $ in doc actually?

File: docs/user-guide/cli-idfplugin.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+At this stage, the Plug-in does not interface with the mainframe system, so no Zowe CLI profile configuration is required.
+
+Use `zowe idf map --help` to get the most up-to-date details of the required parameters.
+
+`zowe idf map "$CSV" --esm "$ESM" --system "$SYSTEM" --registry "$REGISTRY"`
+
+- `$CSV`: Path to the input CSV-formatted file, see below for the details of the format.
+- `$ESM`: Identifier of the target ESM, one of ACF2, TSS or RACF.
+- `$SYSTEM`: Optional parameter, system identifier for JCL purposes.
```
Comment (JirkaAichler) [inline docs/user-guide/cli-idfplugin.md:45]: I would place optional parameters at the bottom (the last position).

Comment (JirkaAichler) [inline docs/user-guide/cli-idfplugin.md:45]: The system must match the system name defined in JES. https://www.ibm.com/docs/en/zos/2.1.0?topic=sp-subparameter-definition-10

Comment (pablocarle) [reply docs/user-guide/cli-idfplugin.md:45]: Thanks, I'll add this information to the doc

Comment (pablocarle) [reply docs/user-guide/cli-idfplugin.md:45]: Agree, thanks

File: docs/user-guide/cli-idfplugin.md (hunk 1 above)
Comment (pablocarle) [reply docs/user-guide/cli-idfplugin.md:46]: I can remove the `$` signs

**Comment by achmelo** [comment]
This PR is based on the master branch but it needs to be based on docs-staging. I have created a new PR based on staging https://github.com/zowe/docs-site/pull/2792

**Review by janan07 — APPROVED** [review]
Reviewed and edited by Tech Info

**Pull request closed without merging**